components/webfield/ConsoleTaskList.js

```diff
@@ -65,11 +65,13 @@ const ConsoleTaskList = ({
           .concat(tagInvitations.map((inv) => ({ ...inv, tagInvitation: true, apiVersion })))
       )
 
+      console.log(allInvitations)
```

Comment:
I think this and the log on line 74 should be removed

Reply:
oops, done.